### DISCUSSION TITLE
Integrating moose based app into our online calculation platform

### DISCUSSION THREAD

**Discussion by Chaoyue-Liu** [idaholab/moose#22406] [Q&A General] [answered]
Hello Moose Team,

We are currently developing an online calculation website which consists of several models developed by our research team. The website is nonprofitable platform, which is open to everyone to access and do the calculation with the model we provide. Because Moose is an open-source code, we are considering to build one of our model with Moose and integrate it to our platform. We want to know if this is under the permission of the Moose license? 

Thank you!

**Comment by GiudGiud** [comment, marked as answer]
@friedmud 
Yes, LGPL 2.1 will allow that

**Reply by Chaoyue-Liu** [reply]
Thank you. Another question, are we obliged to make our app code open source too?

**Reply by GiudGiud** [reply]
LGPL does not enforce that. 

it s the straight GPL that does that.

**Reply by Chaoyue-Liu** [reply]
OK, Thank you!
